### PR TITLE
Add CI/CD workflow based on Github Actions

### PR DESCRIPTION
**Pull request by dvandyk**
- every push to the master branch is checked
 - every pull request is checked
 - every new commit with a tag that start with 'v' is deployed to
   PyPI, if on the repository level two secrets are set up:

   TWINE_USERNAME
   TWINE_PASSWORD

### CODE REVIEW

**Comment by dvandyk** [comment]
@DavidMStraub @peterstangl @jasonaebischerGIT Let me know if this is useful.

File: .github/workflows/check+deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+              uses: actions/checkout@v2
+              with:
+                submodules: true
+                path: _src/
+
+            - name: Install dependencies
+              shell: bash
+              run: |
+                pushd _src
+                python3 -m pip install --upgrade 'numpy>=1.16.5' 'scipy>=1.0' pandas 'pylha>=0.2' pyyaml 'ckmutil>=0.3.2' 'rundec>=0.5' voluptuous nose
```
Comment (DavidMStraub) [inline .github/workflows/check+deploy.yaml:24]: Can't this be skipped? If a dep is missing in setup.py, we wanna catch it

Comment (dvandyk) [reply .github/workflows/check+deploy.yaml:24]: You are absolutely right! This was part of my first attempt, when I did not yet install ``wilson`` using pip, but rather tested in-place with ``nose``. Going to remove it!

Comment (dvandyk) [reply .github/workflows/check+deploy.yaml:24]: Except for nose, which we need for the tests.

**Review by jasonaebischerGIT — APPROVED** [review]

**Comment by jasonaebischerGIT** [comment]
Thanks a lot. This looks very useful!

**Pull request merged**